src/main/java/eu/dissco/backend/controller/BaseController.java

```diff
@@ -26,6 +35,34 @@ public abstract class BaseController {
   protected final ObjectMapper mapper;
   private final ApplicationProperties applicationProperties;
 
+  protected User getUser(Authentication authentication) throws ForbiddenException {
+    var tokenEncoded = ((Jwt) authentication.getPrincipal()).getTokenValue();
+    var tokenBody = tokenEncoded.split("\\.")[1];
+    var tokenString = Base64.getUrlDecoder().decode(tokenBody);
```

Comment:
hmm ok, you probably looked at it but there was no easier way to do it? They aren't properties in a kind of Map or thing in the token...

Reply:
there's actually a way easier way to do this, fixed now